sycl/plugins/level_zero/pi_level0.cpp

```diff
@@ -1834,6 +1835,7 @@ pi_result piProgramRetain(pi_program Program) {
 
 pi_result piProgramRelease(pi_program Program) {
   assert(Program);
+  assert("Program is already released." && Program->RefCount > 0);
```

Comment:
putting the condition first is more readable

Reply:
Done